### DISCUSSION TITLE
Handling tailwind responsiveness

### DISCUSSION THREAD

**Discussion by enyelsequeira** [joe-bell/cva#51] [Q&A] [answered]
Hello,

pretty new to it, so sorry if my question is out place. I’m trying to use tailwind, but my question is how do you handle responsiveness, I know you mentioned that it doesn’t support it the way stitches does and that if we want that we could hide and show between breakpoints, so my question is without doing that is there a way to do it? I was thinking for example passing props to the component itself and the handle the responsiveness with tailwind when the component is actually being used, thoughts?

**Comment by joe-bell** [comment, marked as answer]
Sorry it’s not something I’m planning to offer due to the limitations mentioned in the README

https://github.com/joe-bell/cva#how-can-i-create-responsive-variants-like-stitchesjs

**Reply by enyelsequeira** [reply]
I was thinking something like this, do you think this wouldn't be an option? as of now, of course this doesn't work.

I guess this would be like enhancement for project based and not really the lib, just thinking of ways to achieve it that's all

![image](https://user-images.githubusercontent.com/50872387/198831366-4d064889-faae-48e4-b3b8-ecd4c47d68ff.png)

![image](https://user-images.githubusercontent.com/50872387/198831394-ca9a94f3-190c-4457-bc90-a7edd7d76883.png)
